app/controllers/manuals_controller.rb

```diff
@@ -22,6 +22,19 @@ def new
     @manual = Manual.new
   end
 
+  def create
+    @manual = Manual.new(params["manual"])
+    @manual.organisation_content_ids = current_user.organisation_content_id
```

Comment:
Oh I see... `organisation_content_ids=` calls `Array.wrap`. Hmm this feels a bit surprising to me. I'd favour keeping the setter simple and make sure that the caller passes in a real array...